neuralop/layers/fno_block.py

```diff
@@ -137,14 +137,22 @@ def __init__(
         if norm is None:
             self.norm = None
         elif norm == "instance_norm":
-            self.norm = nn.ModuleList(
-                [
-                    getattr(nn, f"InstanceNorm{self.n_dim}d")(
-                        num_features=self.out_channels
-                    )
-                    for _ in range(n_layers * self.n_norms)
-                ]
-            )
+            if self.n_dim <= 3:
+                self.norm = nn.ModuleList(
+                    [
+                        getattr(nn, f"InstanceNorm{self.n_dim}d")(
```

Comment:
I tink the getattr could be a single line here.
This takes a lot of space and makes it a little hard to read.